lua/octo/completion.lua

```diff
@@ -48,38 +49,51 @@ local function get_repo_issues(repo, params)
 end
 
 function M.issue_complete(findstart, base)
-  -- :he complete-functions
+  -- the complete-functions
   if findstart == 1 then
     -- findstart
     local line = api.nvim_get_current_line()
     local pos = vim.fn.col(".")
-    local i, j = 0
+    local i, j = 0, 0
     while true do
       i, j = string.find(line, "#(%d*)", i + 1)
       if i == nil then
-        break
+        i, j = 0, 0
+        i, j = string.find(line, "@(%w*)", i + 1)
```

Comment:
I couldn't figure out how to get lua's find function to let me do the first regex or the second. This is ugly but seemed to do what I wanted it to do. I'm very open to better ideas here :smile:

Reply:
yep, I dont think thats possible with lua patterns, so the workaround is :+1: